django/api/models/go_electric_rebate_application.py

```diff
@@ -81,6 +81,7 @@ class Status(TextChoices):
     )
     date_of_birth = DateField(validators=[validate_driving_age], null=True)
     tax_year = IntegerField(null=True)
+    approval_email_sent = BooleanField(null=True)
```

Comment:
What's the reasoning behind having a null boolean option here?

Reply:
My thinking was that there are some states an application can be in where it may not make sense for this field to be either true or false; for example, states of "household_initiated", "submitted", "verified" are not yet at the approval/non-approval stage.